<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" verbose="0" name="Failed suite [Suite]" parallel="classes">
  <listeners>
    <listener class-name="com.cpcommunity.ExtentListeners.ExtentListeners"/>
    <listener class-name="com.uiFramework.pamTen.cpcommunity.helper.listener.RetryListener"/>
  </listeners>

  <test name="Test8(failed)" parallel="classes">
    <classes>
      <class name="com.cpcommunity.testScripts.community.TC046_Member_Accept_Community_Request_from_My_Communities">
        <methods>
          <include name="setUpFramework"/>
          <include name="TC041ToTC046" invocation-numbers="0"/>
          <include name="tearDown"/>
          <include name="sendmail"/>
        </methods>
      </class> <!-- com.cpcommunity.testScripts.community.TC046_Member_Accept_Community_Request_from_My_Communities -->
       <class name="com.cpcommunity.testScripts.community.TC075_Member_Cancel_Community_Join_Request_from_My_Communities________">
        <methods>
          <include name="TC075" invocation-numbers="0"/>
          <include name="setUpFramework"/>
          <include name="tearDown"/>
          <include name="sendmail"/>
        </methods>
      </class> <!-- com.cpcommunity.testScripts.community.TC075_Member_Cancel_Community_Join_Request_from_My_Communities________ -->
      
    </classes>
  </test> <!-- Test8(failed) -->
</suite> <!-- Failed suite [Suite] -->
